src/cards/ha-entities-card.html

```diff
@@ -77,6 +77,11 @@
     };
   }
 
+  constructor() {
+    super();
+    this.entityTapped = this.entityTapped.bind(this);
```

Comment:
Can we add a comment here explaining that we need to save a single bound function reference to ensure the event listener can identify it properly?

Reply:
Done

Reply:
👍